documentation/docs/practical-sessions/data-ingest-and-monitoring.md

```diff
@@ -22,154 +33,196 @@ python3 wis2box-ctl.py status
 
-Another method for manually ingesting data is to use the `MinIO` admin interface to upload a file into the `wis2box-incoming` bucket. 
+`xyz/test/data/core/weather/surface-based-observations/synop`
 
-If your data-collection software supports sending data to an FTP endpoint you could use the optional **wis2box-ftp** container setup.
+<img alt="minio-admin-create-new-path" src="../../assets/img/minio-admin-create-new-path.png" width="800">
 
-You can also automate data ingest using a script to copy data into the `wis2box-incoming` bucket at regular intervals, for example using Python and the MinIO-client.
+Upload the file `aws-example.csv` to the directory you just created:
 
-### Download test data
+<img alt="minio-admin-uploaded-file" src="../../assets/img/minio-admin-uploaded-file.png" width="800">
 
-Click on the two links below and download two new data samples on your computer:
+!!! question "Exercise 1: check for errors"
+    Do you see any errors reported on the Grafana dashboard?
 
-<a href="../../sample-data/WIGOS_0-454-2-AWSBILIRA_new.csv">WIGOS_0-454-2-AWSBILIRA_new.csv</a>
+??? note "Click to revel answer"
+    The 'wis2box ERRORs' displayed at the bottom of the Grafana home dashboard should report the following error:    
+    
+    * ERROR - handle() error: Topic Hierarchy validation error: No plugins for http://minio:9000/wis2box-incoming/xyz/test/data/core/weather/surface-based-observations/synop/aws-example.csv in data mappings. Did not match any of the following: ...
 
-<a href="../../sample-data/WIGOS_0-454-2-AWSCHIKANGAWA_new.csv">WIGOS_0-454-2-AWSCHIKANGAWA_new.csv</a>
+    The 'data'-definition in the data-mappings.yml uses . instead of / to separate the path elements.
 
-### MinIO admin interface
+    If there are no data-mappings defined for to the directory that received the data, wis2box will not initiate the workflow.
 
-Open a new tab in your web browser and visit the page `http://<your-host>:9001`. You should see the login screen for MinIO. 
-You can login with username `minio` and password `minio123`:
+!!! question "Exercise 2: correct your input path and repeat the data ingest"
 
-<img alt="minio-admin-buckets" src="../../assets/img/minio-login.png" width="600">
+    Go back to MinIO and to the root of the 'wis2box-incoming' bucket. Then click 'create new path' and define the correct path for wis2box. For example if your country-code is 'idn' and your center-id is 'bmkg', you should create the following path:
 
-You should be see the buckets 'wis2box-archive', 'wis2box-incoming', 'wis2box-public'.
+    * idn/bmkg/data/core/weather/surface-based-observations/synop
+    
+    Now upload the sample data file 'aws-example.csv' to the new path. Do you see any errors reported on the Grafana dashboard?
 
-You can click 'browse' to view the contents of the buckets. 
+??? note "Click to revel answer"
```

Comment:
I've used the "success" admonition for the answers, we probably want to be consistent. I don't mind change what I've done. I may have sprinkled a few "warnings" and "errors" in when I've expected there to be errors in the result.